.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 10.x]
+        sequelize-version: ['5', 'next']
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: run ci
+      run: |
+        npm i sequelize@${{ matrix.sequelize-version }}
+        npm install
+        npm run build
```

Comment:
Interesting, `npm run build` is not present in `.travis-ci.yml`... Is it because the `prepare` hook is ran automatically?

Reply:
Yes, good catch, this is building twice unnecessarily. Have removed.